aten/src/ATen/native/quantized/cpu/qpool.cpp

```diff
@@ -53,10 +52,8 @@ void spatial_dilated_max_pooling(
 
           // local pointers
           T* o_p = oData + p * oW * oH + row * oW + col;
-          int64_t* ind_p = index + p * oW * oH + row * oW + col;
```

Comment:
The float API for nn.maxpool2d is: 
torch.nn.MaxPool2d(kernel_size, stride=None, padding=0, dilation=1, return_indices=False, ceil_mode=False), for the quantized version, can we support return_indices = True with a slower path? This way we can maintain API compatibility with float.